app/views/users/show.html.erb

```diff
@@ -1 +1,32 @@
-<h1>Showing a specific User</h1>
\ No newline at end of file
+<div class="user">
+  <div class="user-photo">
+    <img src="<%= @user.photo %>" alt="User Photo">
+  </div>
+  <div class="user-details">
+    <%= link_to user_url(@user), class: 'name' do %>
+      <%= @user.name %>
+    <% end %>
+    <p class="posts-no">
+      <%= "Number of posts: #{@user.posts_counter}" %>
+    </p>
+  </div>
+</div>
+<div class= "bio">
+    <h3>Bio:</h3>
+    <p><%= @user.bio %></p>
+</div>
+<% @user.recent_posts.each do |post| %>
+     <div class = "post">
+        <%= link_to user_post_url(user_id: @user.id, id: post.id), class: 'post-link' do %>
+          <h3>Post #<%= @user.posts.index(post) + 1 %></h3>
+        <% end %>
+        <h4><%= post.text %></h4>
+          <div class="count">
+            <p>Comments: <%= post.comments_counter %></p>
+            <p>Likes: <%= post.likes_counter %></p>
+          </div>
+     </div>
+<% end %>
+<%= link_to user_posts_url(@user.id), class: 'btn-link' do %>
+  <button>See all posts</button>
+<% end %>
```

Comment:
- [ ] Kindly make sure the views of the app look like the provided wireframe

|Your website|Provided Wirframes|
|-|-|
|![Screenshot_20230629_003537](https://github.com/Odongo006/Rails_blog_App/assets/25227022/f2b52399-f581-400b-84fe-a91509c4ae7f)|![](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)|